ddtrace/tracer.py

```diff
@@ -153,10 +153,20 @@ def _default_span_processors_factory(
     span_processors += [TopLevelSpanProcessor()]
 
     if appsec_enabled:
+        if config._api_security_enabled:
+            from ddtrace.appsec._api_security.api_manager import APIManager
+
+            APIManager.enable()
+
         appsec_processor = _start_appsec_processor()
         if appsec_processor:
             span_processors.append(appsec_processor)
     else:
+        if config._api_security_enabled:
+            from ddtrace.appsec._api_security.api_manager import APIManager
+
+            APIManager.disable()
```

Comment:
Do we need to call disable here?

Reply:
Yes, as APIManager relies on ASM, it can not work properly without it. So if appsec_enabled becomes False, it must be deactivated.